front-end/src/components/cookie/CookieView.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Modal, Button } from "react-bootstrap";
+
+import "./CookieView.css";
+
+class CookieView extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            show: true
+        };
+    
+        this.handleClose = this.handleClose.bind(this);
```

Comment:
you can autobind with this syntax

method = (argList) => {...}

(declaring a method as a property of the class will make it static and bind it to the class itself)

Reply:
Thanks, will check it.